W-13729475 setup desktop duke (#78)
mulesoft/docs-code-builder: modules/ROOT/pages/configure-default-output-panel.adoc

include::reuse::partial$beta-banner.adoc[tag="anypoint-code-builder"]

> Configuring the output panel to display extension logs in *Mule DX Server* enables you to view the status of Anypoint Code Builder activity.  

Author: ```suggestion
Configuring the output panel to display extension logs for *Mule DX Server* enables you to view the status of Anypoint Code Builder activity.  
```

> Configuring the output panel to display extension logs for *Mule DX Server* enables you to view the status of Anypoint Code Builder activity.  

Author: ```suggestion
Configuring the output panel to display logs for the Anypoint Code Builder extensions enables you to view the status of activity in the IDE.  
```

> Configuring the output panel in Anypoint Code Builder to view extension logs in *Mule DX Server* is helpful for understanding the status of Anypoint Code Builder activity.  

Author: ```suggestion
Configuring the output panel to display extension logs in *Mule DX Server* enables you to view the status of Anypoint Code Builder activity.  
```
